OmsAgent/Fairfax/omsagent_ff.py

```diff
@@ -112,6 +113,10 @@ def main():
             message = 'Install failed with exit code {0}: please install ' \
```

Comment:
Just noticed that we are duplicating error codes in extension code . Bundle has already taking care of logging a proper message when given error code is thrown. These logs are available in azure extension logs. please remove duplicate customization from extension code.